testing/tests/destructive.test.js

```diff
@@ -99,8 +99,9 @@ describe("fixing flaws", () => {
       .split("\n")
       .filter((line) => regexPattern.test(line));
     expect(dryRunNotices.length).toBe(3);
-    expect(dryRunNotices[1]).toContain(pattern);
-    expect(dryRunNotices[0]).toContain(path.join(pattern, "images"));
+    expect(dryRunNotices[2]).toContain(pattern);
+    expect(dryRunNotices[1]).toContain(path.join(pattern, "images"));
+    expect(dryRunNotices[0]).toContain(path.join(pattern, "bad_pre_tags"));
```

Comment:
The change to lines 103 and 104 accommodates rename of test folder from `pre_with_html` to `bad_pre_tag` caused by the order of appearance of files in tests.

Prior to rename, order was:
```
.../content/files/en-us/web/fixable_flaws/images/index.html
.../content/files/en-us/web/fixable_flaws/index.html
.../content/files/en-us/web/fixable_flaws/pre_with_html/index.html
```

After the rename:
```
.../content/files/en-us/web/fixable_flaws/bad_pre_tags/index.html
.../content/files/en-us/web/fixable_flaws/images/index.html
.../content/files/en-us/web/fixable_flaws/index.html
```

I also added a check for `dryRunNotices[2]` for readability.